cmd/gpu_plugin/README.md

```diff
@@ -313,6 +316,15 @@ The GPU plugin functionality can be verified by deploying an [OpenCL image](../.
       Warning  FailedScheduling  <unknown>  default-scheduler  0/1 nodes are available: 1 Insufficient gpu.intel.com/i915.
     ```
 
+## Use Cases for Different Modes
+
+Intel GPU-plugin supports a few different operation modes. Depending on the workloads the cluster is running, some modes make less sense than others. Below is a table that explains the differences between the modes and suggests workload types for each mode. The mode selection requires pre-though as it is cluster wide.
+
+| Mode | Sharing | Workload examples | Time critical |
```

Comment:
"examples" may not be best term here, as column describes *all* workloads this mode is intended for.  Maybe "Intended workloads"?

I assume "Time critical" to mean "Whether workloads is latency sensitive".